tools/ci_build/github/apple/build_apple_framework.py

```diff
@@ -66,7 +66,7 @@ def _build_for_apple_sysroot(
         framework_dir = os.path.join(
             build_dir_current_arch,
             build_config,
-            build_config + "-" + sysroot,
+            ((build_config + "-" + sysroot) if sysroot != "macosx" else build_config),
```

Comment:
I think this is not the correct path for static frameworks.
https://github.com/microsoft/onnxruntime/blob/41ea47be1eca597bbe80872d3dbf8a4451ddfd44/cmake/onnxruntime.cmake#L280-L284

perhaps leave it for now and we can fix it fully later.